solr_conf/solrconfig.xml

```diff
@@ -140,7 +140,7 @@
        <str name="facet">true</str>
        <str name="facet.mincount">1</str>
        <str name="facet.limit">10</str>
-       <str name="facet.field">dataProvider_name</str>
+       <str name="facet.field">dataProvider_providedLabel</str>
```

Comment:
The above two lines could be deleted.  I think I forgot to delete them way back when I was learning how solr configuration worked.  They don't hurt anything, either, but I don't think they ever get used.